.pre-commit-config.yaml

```diff
@@ -40,4 +40,8 @@ repos:
       exclude: ^vendor
     - id: no-commit-to-branch
     - id: trailing-whitespace
-      exclude: ^vendor
+      exclude: |
+        (?x)(
+            ^vendor|
+            ^modules/common/util/template_util_test.go
```

Comment:
I had to add this because of the test data for `TestRemoveNewLines` where I test preceding spaces

Reply:
this feels a bit scary. I would the test data to normal strings with \n and \t to avoid this. But that can be done in a follow up.

Reply:
I have removed the spaces from the test data and returned the config.